apps/examples/security_test/security_test_main.c

```diff
@@ -0,0 +1,1597 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * examples/security_test/security_test_main.c
+ *
+ *   Copyright (C) 2016 SAMSUNG ELECTRONICS CO., LTD. All rights reserved.
+ *   Author: Jisuu Kim <jisuu.kim@samsung.com>
+ *
+ *   Copyright (C) 2008, 2011-2012 Gregory Nutt. All rights reserved.
```

Comment:
Nuttx does not have this example. Why do you add Nuttx license here?

Reply:
I didn't check it carefully. Modified.